module/support/optimisation/NSGA2Optimiser/src/random.cpp

```diff
@@ -0,0 +1,32 @@
+#include "random.h"
```

Comment:
in general, I'm not reviewing any of the NSGA stuff... But all this is in the standard library and why are we using boost for this!!